docs/background/faq.rst

```diff
@@ -82,7 +82,7 @@ that caused the problem.
 What version of Python does VOC require?
 ----------------------------------------
 
-VOC runs under Python 3.4+.
+VOC runs under Python 3.4.
```

Comment:
So, even if you can't yet transpile code from 3.5+, you can still run VOC with Python 3.4+.